Docs/12.1

Fweber1331 opened the pull request:
[#40956] Update Docs for 12.1

https://community.openproject.org/work_packages/40956

[CI SKIP]

Updating the docs for OP v12.1.0:
The main changes here are: 1) "user-guide" README.MD table of contents (TOC) was sorted to reflect the order modules actually appear in, in the OP module settings menu. 2) Adding the "Team planner" module to the "user-guide" README.MD TOC. 3) Adding a subdirectory to the "user-guide" directory called "team-planner". 4) Adding and editing the "team-planner" subdirectory README.md file to provide an overview of the team planner module, a "Getting started" tutorial, and explanations of the various functions.

Fweber1331 committed:
Docs changes 12.1

task/40956-update-docs-for-121

Fweber1331 committed:
Update to docs for 12.1

Changed user guide
Established subdirectory "team-planner"
Started editing README for team-planner

Fweber1331 committed:
Docs changes for 12.1

Further edits to README of team-planner subdirectory:

-Intro into functionality
-getting started
-more features explained

Fweber1331 committed:
Update README.md

Fweber1331 committed:
Update readme Team planner

Additional edits to team planner readme file

Fweber1331 committed:
Update README.md

Fweber1331 committed:
Project status README Update for 12.1

Project status update for 12.1 (WP #34708):
-Added new screenshots to reflect 3 add'l status choices: Not Started, Finished, and Discontinued

Directory: docs/user-guide/projects/project-status/REAMDME.md

birthe commented on docs/user-guide/projects/project-settings/modules/README.md:
```diff
@@ -12,6 +12,6 @@ To **activate or deactivate modules** in a project select a project from the sel
 Setting a check mark next to a project module will activate the module in the project and a new menu item will appear in the project menu on the left.
 
-![select-modules-OpenProject](image-20201005151029444.png)
+![image-20220207105240658](image-20220207105240658.png)
```
Please note the comments in the Documentation chanel: Please add an informative image description (and ideally also file name),e.g. [opening the dialog to rename your user account](002-opening-the-dialog-to-rename-your-user-account.png)

Fweber1331 committed:
Calendar Module v12.1 Update

Updated snapshots to most current version, explained "Visibility settings", and introduced/explained multiple calendars, Favorite, Public and Private functions

[#36322] Save different views in calendar module

https://community.openproject.org/work_packages/36322

Fweber1331 committed:
Update to the Team Planner

Update to the Team Planner as well as functionality of the Planning Calendar.

There were several additional features that were added to the Team Planner that were not captured in the last edit, such as the "include projects" filter button.

[#40956] Update Docs for 12.1

https://community.openproject.org/work_packages/40956

Fweber1331 committed:
Use Cases

I added a new folder/repository called "Use Cases". Use cases are frequently asked for how-to scenarios and workarounds. The main README is basically a Table of Contents with links. I have also added 2 use cases, Portfolio Management and Resource Management.

This is is not associated with a specific work package, but rather a value-add to customers

birthe commented on docs/user-guide/projects/project-settings/modules/README.md:
```diff
@@ -12,6 +12,6 @@ To **activate or deactivate modules** in a project select a project from the sel
 Setting a check mark next to a project module will activate the module in the project and a new menu item will appear in the project menu on the left.
 
-![select-modules-OpenProject](image-20201005151029444.png)
+![image-20220207105240658](image-20220207105240658.png)
```
hi @Fweber1331, please make sure to give the image descriptions in [...] a real name, e.g "[activate modules in OpenProject]" and also ideally the image itself, e.g. "(activate-modules.png)".

Fweber1331 replied:
@birthe Got it! Fixed in Calendar, Modules, Team Planner and Use Cases

Fweber1331 committed:
Update Images

Updated images included in initial edits, from randomly generated, numerical file names created by pasting, to more descriptive image names and description.

This was done for updates triggered by v12.1:

- User guide -> Modules (update)
- Team planner (new)
- Calendar (update)
- Use cases (new)

RBernsen commented on docs/user-guide/team-planner/README.md:
```diff
@@ -0,0 +1,99 @@
+4. Team planners with the visibility option "public" unselected appear under the "Private Views" heading in the black menu bar. Note: If "public" is not selected, the calendar is automatically "private" and can only be seen by you, not other members.
+
+   Both "public" and "private" calendars can be added as a "favored" planner, and would appear under the heading "Favorite Views" in the black menu bar.
+
```
https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#ui-text
instead of using inverted commas, please make the terms bold

Fweber1331 replied:
Thank you, Rebecca! Changes submitted.

RBernsen reviewed (commented):
Hi Frank! I only had a look at the team planner section, not too sure if that is enough (if not, please let me know...!). The only thing I found was the formatting, insetad of inverted conas, we are using bold: https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#ui-text
Rest looks fine to me.
Thanks a lot

Fweber1331 committed:
Style guide related changes

Style Guide related changes/adjustments to:

- Docs/User Guide/Team Planner
- Docs/User Guide/Calendar
- Docs/Use Cases/*

Detail: Corrected inverted quotation marks use for buttons, and replaced by bold font instead

as-op committed:
Merge branch 'dev' into docs/12.1

# Conflicts:
#	docs/user-guide/projects/project-settings/modules/image-20201005151029444.png

as-op committed:
fix(docs-12.1): typofix in name & url for new section

as-op commented on docs/use-cases/portfolio-management/README.md:
```diff
@@ -0,0 +1,44 @@
+---
+sidebar_navigation:
+  title: OpenProject Use-Cases: Portfolio Management and Custom Reporting
```
Please use only short text for the title of entry in the sidebar, so the entries don't span in multiple lines.
e.g. "Portfolio Management and Custom Reporting" is enough.

as-op commented on docs/use-cases/portfolio-management/README.md:
```diff
@@ -0,0 +1,44 @@
+  priority: 999
+description: Step-by-step instruction about portfolio management and custom reporting
+robots: index, follow
+keywords: help, documentation, use-cases, portfolio management
```
Please don't use "help" and "documentation" as keywords for specific topic pages. (Only the root page should have them).

as-op commented on docs/use-cases/portfolio-management/README.md:
```diff
@@ -0,0 +1,44 @@
+---
+# Use Cases:
+
+# Portfolio Management and Custom Reporting Options
```
Please don't use multiple level-1 headlines after another, better combine them 
`# Use Cases: Portfolio Management and Custom Reporting Options`

as-op commented on docs/use-cases/README.md:
```diff
@@ -0,0 +1,18 @@
+
+| Use Case                                      | Detailed Description                                         |
+| --------------------------------------------- | ------------------------------------------------------------ |
+| [Portfolio Management](/portfolio-management) | This guide provides detailed step-by-step instruction on how to set up an overview of your project portfolio and create custom reports using the Project Overview, Wiki and the Rich text (WYSIWYG) editor in OpenProject. |
```
Unfortunately `[Portfolio Management](/portfolio-management)` on the website would be expanded to `www.openproject.org/portfolio-management` not `www.openproject.org/docs/use-cases/portfolio-management`
You can use the relative dot notation `[Portfolio Management](./portfolio-management)` 
or if it's a children page as in this case you can omit the path `[Portfolio Management](portfolio-management)`

as-op commented on docs/use-cases/portfolio-management/README.md:
```diff
@@ -0,0 +1,44 @@
+
+ 
+
+Step 2: You can sort and filter this view using custom fields, such as *Initiative* or *Status.* If you have not added useful custom fields, please see: https://www.openproject.org/docs/system-admin-guide/custom-fields/
```
Please use the markdown link format [title](relative path).
`[Custom fields](../../system-admin-guide/custom-fields/)`

as-op commented on docs/user-guide/team-planner/README.md:
```diff
@@ -0,0 +1,101 @@
+| [Step 1: Adding team members](team-planner) | How to add team members? |
+| [Step 2: Adding/working with work package strips](team-planner) | How to work with work package strips? |
+| [Work package strip detail view](team-planner) | How to access the work package detail view? |
+| [Removing a work package strip](team-planner) | How to remove a work package strip? |
```
The anchor links in this table are missing. e.g. `[Removing a work package strip](#removing-a-work-package-strip)`

as-op commented on docs/user-guide/team-planner/README.md:
```diff
@@ -0,0 +1,101 @@
+
+
+
+- **Removing a work package strip**
```
Please use headlines format, manually boldend text can't be linked to. e.g. for Level-2 headline
 `## Removing a work package strip` the anchor-id `#removing-a-work-package-strip` is auto generated, so we can link to this section.

as-op commented on docs/user-guide/team-planner/README.md:
```diff
@@ -0,0 +1,101 @@
+
+## Getting started
+
+- ##### Step 1: Adding team members
```
Please don't skip headline levels & remove the minus in front which indicates a (simple) markdown list
```

## Getting started

### Step 1: ...
 ....
### Step 2: ...
 ...

```

as-op reviewed (changes requested):
Hi, unfortunately I found some issues. Please call me if you have any questions.

as-op committed:
fix(docs12.1): resolve review issues

as-op committed:
fix(docs12.1): use relative links for internal doc links
